### PR TITLE
Add Support for autopurge flags

### PR DESCRIPTION
**Pull request by jonathanl224**
Added Support for Parameters autopurge.snapRetainCount and
autopurge.purgeInterval included Default Values and README Update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,17 @@ Defaults to `false`. Zookeeper's [`standaloneEnabled`](http://zookeeper.apache.o
 
 > Prior to 3.5.0, one could run ZooKeeper in Standalone mode or in a Distributed mode. These are separate implementation stacks, and switching between them during run time is not possible. By default (for backward compatibility) standaloneEnabled is set to true. The consequence of using this default is that if started with a single server the ensemble will not be allowed to grow, and if started with more than one server it will not be allowed to shrink to contain fewer than two participants.
 
+### `ZOO_AUTOPURGE_PURGEINTERVAL`
+
+Defaults to `0`. Zookeeper's [`autoPurge.purgeInterval`](https://zookeeper.apache.org/doc/trunk/zookeeperAdmin.html#sc_advancedConfiguration)
```
Comment (31z4) [inline README.md]: The link is dead.

File: README.md [hunk 2]
```diff
@@ -115,6 +115,17 @@ Defaults to `false`. Zookeeper's [`standaloneEnabled`](http://zookeeper.apache.o
...
+> New in 3.4.0: The time interval in hours for which the purge task has to be triggered. Set to a positive integer (1 and above) to enable the auto purging. Defaults to 0.
+
+### `ZOO_AUTOPURGE_SNAPRETAINCOUNT`
+
+Defaults to `3`. Zookeeper's [`autoPurge.snapRetainCount`](https://zookeeper.apache.org/doc/trunk/zookeeperAdmin.html#sc_advancedConfiguration)
```
Comment (31z4) [inline README.md]: The link is dead.

**Comment by 31z4** [comment]
Thanks for you PR @jonathanl224 👍  Do these options work as expected for you? Could you please help figuring out what could be wrong in #30 🙏

File: README.md [hunk 3]
```diff
@@ -111,10 +111,21 @@ Defaults to `60`. ZooKeeper's `maxClientCnxns`
 
 ### `ZOO_STANDALONE_ENABLED`
 
-Defaults to `false`. Zookeeper's [`standaloneEnabled`](http://zookeeper.apache.org/doc/trunk/zookeeperReconfig.html#sc_reconfig_standaloneEnabled)
+Defaults to `false`. Zookeeper's [`standaloneEnabled`](http://zookeeper.apache.org/doc/current/zookeeperReconfig.html#sc_reconfig_standaloneEnabled)
```
Comment (31z4) [inline README.md]: The link is dead again. Must be `r3.5.3-beta` instead of `current`.

File: README.md [hunk 4]
```diff
@@ -127,7 +138,7 @@ The id must be unique within the ensemble and should have a value between 1 and
 
 This variable allows you to specify a list of machines of the Zookeeper ensemble. Each entry has the form of `server.id=host:port:port`. Entries are separated with space. Do note that this variable will not have any effect if you start the container with a `/conf` directory that already contains the `zoo.cfg` file.
 
-In 3.5, the syntax of this has changed. Servers should be specified as such: `server.id=<address1>:<port1>:<port2>[:role];[<client port address>:]<client port>` [Zookeeper Dynamic Reconfiguration](http://zookeeper.apache.org/doc/trunk/zookeeperReconfig.html)
+In 3.5, the syntax of this has changed. Servers should be specified as such: `server.id=<address1>:<port1>:<port2>[:role];[<client port address>:]<client port>` [Zookeeper Dynamic Reconfiguration](https://zookeeper.apache.org/doc/current/zookeeperAdmin.html)
```
Comment (31z4) [inline README.md]: The link is dead again. Must be https://zookeeper.apache.org/doc/r3.5.3-beta/zookeeperReconfig.html

**Comment by eyalzek** [comment]
Any way to rebase and merge? I can submit a fresh PR if this is abandoned.

**Comment by 31z4** [comment]
Closing in favor of 
https://github.com/31z4/zookeeper-docker/pull/40

**Pull request closed without merging**